Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM puppet/pdk:latest
 USER root
 
 RUN apt-get update && \
-    apt-get install -y curl && \
+    apt-get install -y curl jq && \
```

Comment:
If we go the route of making a dedicated Dockerfile for the malware scanner, we wouldn't need to add `jq` here - but, I think we could simplify some of our existing code by using `jq`, so it's probably a worthwhile addition either way.